scripts/Config.ts

```diff
@@ -2,12 +2,11 @@ import { ethers } from 'ethers';
 
 const config = {
   /// Volt System Oracle Params
-  STARTING_ORACLE_PRICE: '1054710229549539283', /// pay out 92 bips for July 15th through August 2nd
-  ORACLE_PERIOD_START_TIME: '1659466800', /// August 2nd start time
-  MONTHLY_CHANGE_RATE_BASIS_POINTS: 5, /// AAVE Trailing 30 Day Average yield is 60 bips per year
+  STARTING_ORACLE_PRICE: '1055476736004932282',
+  ORACLE_PERIOD_START_TIME: '1663286400', /// September 13th, 12pm pst start time
+  MONTHLY_CHANGE_RATE_BASIS_POINTS: 12, /// Compound Trailing 30 Day Average DAI yield is ~1.5% per year, 150 bips
```

Comment:
Seems reasonable. 12*12 = 144 without compounding. Remind me again - there's compounding at the end of each month right? So this won't actually be 144 bips annually, but a bit more?

Reply:
correct, it will be a bit more than 1.44% annually due to compounding